actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -46,6 +46,22 @@ def render
       options = @options.stringify_keys
       add_default_name_and_id(options)
       options["input"] ||= dom_id(object, [options["id"], :trix_input].compact.join("_")) if object
+
+      translate_label = options.delete("label")
+
+      if !options.key?("aria-label") && translate_label
```

Comment:
To me, `label: true` does not communicate ARIA-related behavior.  What if this was activated by `options["aria-label"] == true` instead?